index.js

```diff
@@ -297,10 +299,28 @@ Projects();
 
 inTouch.addEventListener('click', () => {
   const emailValue = email.value;
+  const titleValue = title.value;
+  const descriptionValue = description.value;
+
   if (emailValue.trim() === '' || emailValue !== emailValue.toLowerCase()) {
     email.setCustomValidity('Please supply a valid email address in lower case');
     email.reportValidity();
   } else {
     email.setCustomValidity('');
   }
+
+  const userInput = {
+    title: titleValue,
+    email: emailValue,
+    description: descriptionValue,
+  };
+
+  localStorage.setItem('userInfo', JSON.stringify(userInput));
+});
+
+const getValue = JSON.parse(localStorage.getItem('userInfo'));
+document.addEventListener('DOMContentLoaded', () => {
+  title.value = getValue.title;
+  email.value = getValue.email;
+  description.value = getValue.description;
```

Comment:
According to the project requirements, after filling out the form, and not submitting it, the entered values should be saved in the browser. Your data is only preserved upon form submission.

- Kindly make it so that data is saved `on value change` rather than on form submission.